books/free-programming-books-tr.md

```diff
@@ -110,6 +110,11 @@
 * [Java Kitabı](https://ia601503.us.archive.org/27/items/java-kitabi/java-kitabi.pdf) (PDF)
 
 
+### JavaScript
```

Comment:
need to add this to the index at the top of page

Reply:
Sorry missed that somehow. Anyways fixed it!